### DISCUSSION TITLE
missing dSYM

### DISCUSSION THREAD

**Discussion by raenk** [codemagic-ci-cd/codemagic-docs#1954] [Q&A] [answered]
### If you are looking for help, please confirm the following...

- [X] I confirm I have searched the [Docs](https://docs.codemagic.io), [Codemagic Sample Projects](https://github.com/codemagic-ci-cd/codemagic-sample-projects), and GitHub [Discussions](https://github.com/orgs/codemagic-ci-cd/discussions).
- [X] I confirm the software versions on my local computer match the ones in CI environment (e.g. Xcode version, Gradle version). See [here](https://docs.codemagic.io/troubleshooting/common-android-issues/#builds-work-locally-but-fail-on-codemagic)
- [X] I confirm I am able to do a fresh clone of the repository and build the project on my local computer.

### Which mobile framework are you using?

Flutter (Dart)

### Steps to reproduce

I follow the codemagic docs: '[How to upload dSYM artifacts to Firebase Crashlytics using Workflow Editor](https://docs.codemagic.io/knowledge-firebase/firebase-crashlytics-dsym-uploading/)'

However, Crashlytics still complains about missing dSYM.

### Expected results

dSYM correctly uploaded and no complaints from firebase crashlytics.

### Actual results

Crashlytics warns/notifies me about missing dSYM everytime.

### Build id (optional)

64d2f5794f658a2519a73e20

**Comment by himesh-cm** [comment, marked as answer]
Hi @raenk, Can you remove the `head -1` from the script where `dsymPath` is defined? This will allow you to upload all the files.
Please let us know, how it goes.

**Comment by raenk** [comment]
It worked! Thanks @himesh-cm 
I wonder if the docs are outdated (?) or the `head -1` works for specific cases.

**Reply by himesh-cm** [reply]
Hi @raenk, It is updated. It depends on your framework and what `dSYM` file you need.
While I’m closing our discussion for now, if you have any other questions or suggestions, feel free to reply back or start a new discussion anytime. Have a nice day!
